manifests/e/Evope/Evope/1.1.3.94/Evope.Evope.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=AUSU.CRLF.5-1-19041-2364.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Evope.Evope
+PackageVersion: 1.1.3.94
+InstallerType: inno
+Installers:
+- Architecture: x64
+  InstallerUrl: https://app.evope.com.br/evopesetup.exe
+  InstallerSha256: E804D4FB260B6BCB81C675C6A020284DC4961A650474CBE0AC6F7E808A933AEE
```

```suggestion
  InstallerSha256: F35544F1375F11A89878E5BC9E85295EB4B2551B6E59EABCA7354B0ADC725AE8
```
Maybe a CDN-cache issue? I see this hash on my end

---

Must be a CDN cache, since I used YamlCreate to update it, so I know the hash on my end is what I got.

---

I also see the hash "F35...AE8"

---

Looks like it just updated on my end too